### ISSUE TITLE
Allow merging of accounts

### ISSUE THREAD

**Issue by ralphbean** [fedora-infra/fmn#9]
After #8, we allowd a plurality of openid providers.

Some people in the future might want to associate their two accounts.  We should allow that but we'll need to "merge" two existing accounts, keep multiple openids on one account, discard one of the two accounts as the "old" one.  it'll be tricky, I think, but cool.

**Comment by ralphbean** [comment]
Note, this doesn't really apply to our Fedora Infra hosted instance.  We disable all other login types except Fedora.  This is now just a 'nice to have' feature for other people who might want to deploy their own FMN.

**Comment by ralphbean** [comment]
I don't think I'm going to bother implementing this unless someone actually asks for it some day.
